pkg/errors/error.go

```diff
@@ -54,3 +54,21 @@ func NewInternalError(err error, details string) *Error {
 		Details: details,
 	}
 }
+
+func NewNotFoundError(err error, details string) *Error {
+	return &Error{
+		Status:  http.StatusText(http.StatusNotFound),
+		Code:    http.StatusNotFound,
+		Message: err.Error(),
+		Details: details,
+	}
+}
+
+func NewBadRequest(message, details string) *Error {
```

Comment:
Please add a test for this one too.

Reply:
updated here: 3cd7858e9204ccb4210d579bd1d341567ee9a2b0